### ISSUE TITLE
 jupyter nbextension configurator tab gives 404

### ISSUE THREAD

**Issue by kklepper** [Jupyter-contrib/jupyter_nbextensions_configurator#98]
Sorry to come back to this issue:

@juhasch
Contributor
juhasch commented Jan 6, 2019

> Should be fixed in version 0.4.1. If you still have problems after the update, please open a new issue.

Still no menu entry, no tab 

http://localhost:8888/nbextensions

    404 : Not Found
    You are requesting a page that does not exist!

```
D:\data\Jupyter>jupyter --version
4.4.0

D:\data\Jupyter>pip show jupyter_nbextensions_configurator
Name: jupyter-nbextensions-configurator
Version: 0.4.1
Summary: jupyter serverextension providing configuration interfaces for nbextensions.
Home-page: https://github.com/jupyter-contrib/jupyter_nbextensions_configurator
Author: jcb91, jupyter-contrib developers
Author-email: joshuacookebarnes@gmail.com
License: BSD 3-clause
Location: c:\users\karl\appdata\local\programs\python\python36\lib\site-packages
Requires: notebook, pyyaml, jupyter-contrib-core, tornado, traitlets, jupyter-core
Required-by: jupyter-contrib-nbextensions

D:\data\Jupyter>jupyter nbextension enable varInspector/main
Enabling notebook extension varInspector/main...
      - Validating: ok

D:\data\Jupyter>jupyter nbextensions_configurator enable --system
Enabling: jupyter_nbextensions_configurator
- Writing config: C:\ProgramData\jupyter
    - Validating...
      jupyter_nbextensions_configurator 0.4.1 ok
Enabling notebook nbextension nbextensions_configurator/config_menu/main...
Enabling tree nbextension nbextensions_configurator/tree_tab/main...

D:\data\Jupyter>jupyter contrib nbextension install --sys-prefix
...
[I 18:55:12 InstallContribNbextensionsApp] - Validating: ok
[I 18:55:12 InstallContribNbextensionsApp] Installing jupyter_contrib_nbextensions items to config in c:\users\karl\appdata\local\programs\python\python36\etc\jupyter
Enabling: jupyter_nbextensions_configurator
- Writing config: c:\users\karl\appdata\local\programs\python\python36\etc\jupyter
    - Validating...
      jupyter_nbextensions_configurator 0.4.1 ok
Enabling notebook nbextension nbextensions_configurator/config_menu/main...
Enabling tree nbextension nbextensions_configurator/tree_tab/main...
[I 18:55:12 InstallContribNbextensionsApp] Enabling notebook extension contrib_nbextensions_help_item/main...
[I 18:55:12 InstallContribNbextensionsApp]       - Validating: ok
[I 18:55:12 InstallContribNbextensionsApp] - Editing config: c:\users\karl\appdata\local\programs\python\python36\etc\jupyter\jupyter_nbconvert_config.json
[I 18:55:12 InstallContribNbextensionsApp] --  Configuring nbconvert template path
[I 18:55:12 InstallContribNbextensionsApp] --  Configuring nbconvert preprocessors
[I 18:55:12 InstallContribNbextensionsApp] - Writing config: c:\users\karl\appdata\local\programs\python\python36\etc\jupyter\jupyter_nbconvert_config.json
[I 18:55:12 InstallContribNbextensionsApp] --  Writing updated config file c:\users\karl\appdata\local\programs\python\python36\etc\jupyter\jupyter_nbconvert_config.json
```

Any more info needed? What can I do?

**Comment by kklepper** [comment]
Resolved the issue with a simple 

    conda install -c conda-forge jupyter_contrib_nbextensions

Could not use conda before because, on Windows, you have to update PATH manually by adding 

    C:\Users\USER_NAME\Anaconda3;C:\Users\USER_NAME\Anaconda3\Library\mingw-w64\bin;C:\Users\USER_NAME\Anaconda3\Library\usr\bin;C:\Users\USER_NAME\Anaconda3\Library\bin;C:\Users\USER_NAME\Anaconda3\Scripts;

To do this, see [How to set the path and environment variables in Windows](https://www.computerhope.com/issues/ch000549.htm)

**Comment by kklepper** [comment]
I noticed lots of errors when starting the server via 

    C:\Users\KARL\Anaconda3\python.exe C:\Users\KARL\Anaconda3\cwp.py C:\Users\KARL\Anaconda3 C:\Users\KARL\Anaconda3\python.exe C:\Users\KARL\Anaconda3\Scripts\jupyter-notebook-script.py D:\data\Jupyter

(This was an option in the Anaconda menu I had before I installed the latest 64-bit version)

All errors have the same syntax; e.g. 

    [W 21:29:23.340 NotebookApp] [jupyter_nbextensions_configurator] nbextension 'skip-traceback/main' has duplicate listings in both 'C:\\Users\\KARL\\Anaconda3\\share\\jupyter\\nbextensions\\skip-traceback\\skip-traceback.yaml' and 'C:\\Users\\KARL\\AppData\\require_once();

`C:\Users\KARL\Anaconda3\share\jupyter\nbextensions\skip-traceback\skip-traceback.yaml` does not mention `main`.

 `C:\Users\KARL\AppData\require_once();` does not make sense to me -- and the directory `C:\Users\KARL\AppData\` is empty

Anyway the errors don't seem to play any role. Do you have an explanation? Most probably my different attempts to install `nbextensions` have caused this. How to get rid of them, then?

**Comment by kklepper** [comment]
To be sure, I uninstalled all Python/Anaconda stuff I had on my machine and installed Anaconda / Rise / nbextensions from scratch. Same result.

**Comment by kklepper** [comment]
BTW: from [Uninstalling Anaconda](https://docs.anaconda.com/anaconda/install/uninstall/)

> Option A. Use simple remove to uninstall Anaconda:
>
>    Windows
>            Use Windows Explorer to delete the envs and pkgs folders prior to running the uninstall in the root of your installation.
>            In the Control Panel, choose Add or Remove Programs or Uninstall a program, and then select Python 3.6 (Anaconda) or your version of Python.

I spent quite some time to make sense of  `envs and pkgs folders` but couldn't. What do they mean? Maybe this is the source of the problem, but if so, this remark is not helpful at all.

**Comment by kklepper** [comment]
Thanks for listening. 

I reinstalled the latest Anaconda and found those directories are subfolders of the main folder. 

Also, the error message or rather warning I posted has all the information I needed, i.e. the two  identical directories from different installations. Renaming or deleting the old solved the issue.
